build_all/linux/build.sh

```diff
@@ -47,7 +52,11 @@ usage ()
     echo " --no-logging                  Disable logging"
     echo " --provisioning                Use Provisioning with Flow"
     echo " --use-tpm-simulator           Build TPM simulator"
-    echo " --use-edge-modules            Build Edge modules"    
+    echo " --use-edge-modules            Build Edge modules" 
+    echo " --use-hsmsymmkey              Build with HSM for Symmetric Key" 
+    echo " --use-hsmsas                  Build with HSM for TPM"  
```

Comment:
should it be tpm?

Reply:
Unfortunate naming: it's all over CMake scripts.

TPM is SAS based and, since originally DPS supported TPM (SAS + asymmetric key provisioning) and X509, it ended up being called "SAS".
When we've added non-TPM asymmetric provisioning Shared Access Key, we called that one Symmetric Key.